js/grid/generate_grid.js

```diff
@@ -91,6 +91,11 @@ function appendTableRows(timetableTableFall, timetableTableSpring) {
         appendTableData(trFall, trSpring, i);
         timetableTableSpring.append(trSpring);
         timetableTableFall.append(trFall);
+        var trFall = $('<tr></tr>');
+        var trSpring = $('<tr></tr>');
```

Comment:
`trFall` and `trSpring` were declared above. There is no need to use `var` here.